apps/client/components/ui/PrimaryButton.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 export const PrimaryButton = ({ href, title }: Props) => {
   return (
```

Comment:
Tu tak samo jak wyżej - nazwa + title (tak samo jak wyżej) to trochę głupia nazwa po za tym to powinno być children ;)